README.md

```diff
@@ -1,2 +1,16 @@
 # sc-ewon-flexy-common-docs
 Repository for common documentation used in Ewon Flexy projects
+
+## Usage
+This repo is intended to be used as a Git submodule in an Ewon Flexy Java project utilizing the [Solution Center Ewon Flexy Extensions Library](https://github.com/hms-networks/sc-ewon-flexy-extensions-lib) with Docusaurus based documentation.
+
+## Requirements
+
+### ScDocusaurusConfig.js
+A `ScDocusaurusConfig.js` must exists in the `web-docs` directory and must match the format defined in the sc-java-maven-starter-project [ScDocusaurusConfig.js](https://github.com/hms-networks/sc-java-maven-starter-project/blob/main/web-docs/ScDocusaurusConfig.js) file.
+
+## Installation
+```console
+cd web-docs/docs
+git submodule add https://github.com/hms-networks/sc-ewon-flexy-common-docs
+```
```

Comment:
Shouldn't the README.md contents be updated to reflect the expected usage of this repository via the Docusarus plugin?

Reply:
yes, it would help if i saved that file.

Reply:
the installation and usage section have been removed entirely. Since the starter project is what will "implement" usage I dont think this needs to be documented here.